File: EasyChina_App/android/EasyChina/app/src/main/java/com/easychina/app/AppConfig.java
package com.easychina.app;

import com.easychina.app.BuildConfig;

public class AppConfig {
    public static final String USER_WEB_URL = BuildConfig.DEBUG
        ? "http://10.0.2.2:5173"  // Android emulator -> host
        : "https://easychina.app";

    public static final String API_BASE_URL = BuildConfig.DEBUG
        ? "http://10.0.2.2:8000"
        : "https://api.easychina.app";

    public static final String AMAP_KEY = "YOUR_AMAP_ANDROID_KEY";
}

File: EasyChina_App/ios/EasyChina/EasyChina/AppConfig.swift
import Foundation

struct AppConfig {
    #if DEBUG
    static let userWebURL = URL(string: "http://localhost:5173")!
    static let apiBaseURL = "http://localhost:8000"
    #else
    static let userWebURL = URL(string: "https://easychina.app")!
    static let apiBaseURL = "https://api.easychina.app"
    #endif

    static let amapKey = "YOUR_AMAP_IOS_KEY"
}
